docs/quickstart.md

```diff
@@ -100,7 +157,7 @@ It's easy to see what has been released using Helm:
 ```console
 $ helm ls
 NAME             VERSION   UPDATED                   STATUS    CHART
```

Comment:
I think the indentation here will also have to change to match the output below.

Reply:
Good catch, fixed. :metal: